modules/system/classes/MailManager.php

```diff
@@ -93,21 +96,73 @@ public function addContentToMailer($message, $code, $data, $plainOnly = false)
             return false;
         }
 
-        $this->addContentToMailerInternal($message, $template, $data, $plainOnly);
+        $html = $text = $template;
+
+        if ($plainOnly) {
+            $html = null;
+        }
+        $this->addContentToMailerInternal($message, $html, $text, $data);
 
         return true;
     }
 
     /**
-     * Internal method used to share logic between `addRawContentToMailer` and `addContentToMailer`
+     * Restore proper behavior in-line with Laravel Mailer
+     * Replaces both `addContentToMailer` and `addRawContentToMailer`
      *
      * @param \Illuminate\Mail\Message $message
-     * @param string $template
+     * @param string $view
+     * @param string $plain
+     * @param string $raw
      * @param array $data
-     * @param bool $plainOnly Add only plain text content to the message
      * @return void
      */
-    protected function addContentToMailerInternal($message, $template, $data, $plainOnly = false)
+    public function addContent($message, $view, $plain, $raw, $data)
```

Comment:
@mjauvin I imagine most of these parameters will be optional, since not all types of content will be defined every time.
```suggestion
    public function addContent($message, $view = null, $plain = null, $raw = null, $data = [])
```

Reply:
Actually, they're always provided, even if null.

Reply:
@mjauvin If that were the case, we wouldn't need the `isset` checks throughout this method. We also can't discount that perhaps someone is using this method directly, as opposed to passing it through the event.